src/search/landmarks/landmark_cost_assignment.h

```diff
@@ -59,12 +60,11 @@ class LandmarkEfficientOptimalSharedCostAssignment : public LandmarkCostAssignme
 
     lp::LinearProgram build_initial_lp();
 public:
-    LandmarkEfficientOptimalSharedCostAssignment(
-        const std::vector<int> &operator_costs,
-        const LandmarkGraph &graph,
-        lp::LPSolverType solver_type);
+    OptimalCostPartitioningAlgorithm(const std::vector<int> &operator_costs,
+                                     const LandmarkGraph &graph,
+                                     lp::LPSolverType solver_type);
 
-    virtual double cost_sharing_h_value(
+    virtual double compute_cost_partitioning(
```

Comment:
Rename to `compute_cost_partitioned_h_value`? Not sure.

Reply:
I added the change. I'm also not sure what would be better, though.

Reply:
I'm not sure we're 100% consistent with this, but we use `compute_...` for void methods that store their results as a side effect because "compute" doesn't imply that something is passed back, only that a computation happens. This other common type of method is one with no side effects, but returning a value, for which we usually use words like `get_...`. We tend not to have methods both have a side effect and return a value, although I'm not sure in this case because the method is non-const. But there is also the distinction between having no side effects in a stronger sense or having no externally visible side effects.

Long story short, perhaps we want `get_cost_partitioned_h_value`.

Reply:
...and perhaps write "heuristic", not "h".